deploy/terraform/network.tf

```diff
@@ -178,17 +193,17 @@ resource "aws_security_group" "nia_gp2gp_mq" {
 
 }
 
-resource "aws_security_group" "ps_db_migration" {
-  name        = "ps_db_migration"
+resource "aws_security_group" "ps_db_sg" {
+  name        = "ps_db_sg"
   description = "Security group for db migration"
-  vpc_id = aws_vpc.nia_gp2gp_vpc.id
+  vpc_id      = aws_vpc.nia_gp2gp_vpc.id
 
   ingress {
     description     = "Allow PS DB traffic"
-    from_port       = "5432"
+    from_port       = var.PS_DB_PORT
     protocol        = "tcp"
-    to_port         = "5432"
-    security_groups = [aws_security_group.nia_gp2gp_dmz.id]
+    to_port         = var.PS_DB_PORT
+    cidr_blocks = ["0.0.0.0/0"]
```

Comment:
Is allowing connectivity from anywhere advisable?

Reply:
yes, but this doesn't work (the connection from aws_security_group.nia_gp2gp_dmz.id) with this configuration hence I had to change it all though it is not open to external connections, I tested it. For DB we are using private subnet so it doesn't mean that it is open to all.

Reply:
this is a place for further deeper analysis but at the moment it seems to be working OK now, the DB is not accessible for external access.

Reply:
I can add an additional ticket to see how this cidr "0.0.0.0/0" block can be more limited to connections from nia_gp2gp_dmz security group only.